docs/03_tutorials/20_tic-tac-toe-game-smart-contract-single-node.md

```diff
@@ -0,0 +1,765 @@
+
+In the example below the player who placed x has won.
+
+| Row       | 0     | 1     | 2     |
+| :-------: | :---: | :---: | :---: |
+|     0     |   -   |   o   |   x   |
+|     1     |   -   |   x   |   -   |
+|     2     |   x   |   o   |   o   |
+
+When all the squares contain a marker and no player has three markers in a row, then the game is a draw.
+
+### Accounts, key pairs, and secure your private key
+Accounts are stored on the blockchain with a public key. Use a private key to access the account, though be sure to secure your private key. For more information about account and permissions click on this link [Accounts and Permissions.](https://developers.eos.io/welcome/latest/protocol/accounts_and_permissions "Accounts and Permissions Overview")
+
+### Smart Contract Actions
+A smart contract exposes methods or ‘actions’ that transactions use to operate the game logic. Transactions may contain one or more ‘actions’. Transactions are generated dynamically outside the smart contract, within an application, or from the command line to call smart contract actions and execute business logic within a smart contract. Transactions are atomic. For example, if one action of a transaction fails the entire transaction fails and the blockchain state is restored to the original state. For more details about transactions and actions click on this link [Transaction Protocol.](https://developers.eos.io/welcome/latest/protocol/transactions_protocol "Tranasctions Protocol") You can use `cleos` to create transactions and push transactions to the blockchain. Transactions contain one or more actions. You can also use `cleos` to call actions directly. Actions can call other actions and can also call actions from other smart contracts.
+
+![Transactions and Actions](./images/tictactoe/transactions-and-actions-temp.png "Transactions and Actions")
+
+### EOSIO Resources 
+The EOSIO blockchain accounts own and consume three resources. By default a single node testnet does load the smart contract which tracks resources therefore this section is for information only:
+
+* RAM - This resource is the amount of RAM available to an account. RAM stores smart contracts loaded on the blockchain. Smart contacts use RAM via the multi-index table to store smart contract state. Spend tokens to purchase more RAM, RAM can be sold.
+* CPU - This resource is the amount of CPU which can be used by an account in each 24 hour period. Transactions consume CPU. Stake tokens for more CPU. Unstaking returns CPU. 
+* NET - This resource is the amount of data which can be written to blockchain blocks by an account in each 24 hour period. The blockchain blocks store transactions and transaction parameters, the history of blockchain . Stake tokens for more NET. Unstaking returns NET.
+
+For more information click on this link [Core Concepts](https://developers.eos.io/welcome/latest/overview/core_concepts)
+
+## Create the necessary accounts and key pairs
```

Comment:
Long heading (getting truncated).

Suggestion:

## Create Accounts and Key Pairs